### ISSUE TITLE
WebID predicate is missing from some accounts

### ISSUE THREAD

**Issue by srosset81** [activitypods/activitypods#323]
On mypod.store, the users `effel` and `nandi` (which are rather new users) had no `semapps:webId` predicate attached to their account (in the settings dataset).

This crashed the code to load notifications channels. I improved this code [here](https://github.com/assemblee-virtuelle/semapps/commit/60a3825d8250a657f50120f8bce2e5ec656bd402) but we need to find out how it is possible for an account to be without a webId.

**Comment by srosset81** [comment]
This also happened to the following users, which are also new users:

- `fusion`
- `pcgamehunter`
- `chrisfr`
- `admin`

5 of these users (including `effel`) registered at the same moment.

**Comment by srosset81** [comment]
After adding the webId manually to these accounts, I had this error when calling

```
[18:00:10.796Z] ERROR SOLID-NOTIFICATIONS.PROVIDER.WEBHOOK: Could not load channels for https://mypod.store/pcgamehunter. Error: The dataset pcgamehunter doesn't exist
[18:00:13.159Z] ERROR SOLID-NOTIFICATIONS.PROVIDER.WEBHOOK: Could not load channels for https://mypod.store/effel. Error: Container not found https://mypod.store/effel/data/notify/webhook-channel2023 (webId https://mypod.store/effel / dataset effel)
[18:00:15.039Z] ERROR SOLID-NOTIFICATIONS.PROVIDER.WEBHOOK: Could not load channels for https://mypod.store/fusion. Error: The dataset fusion doesn't exist
[18:00:15.085Z] ERROR SOLID-NOTIFICATIONS.PROVIDER.WEBHOOK: Could not load channels for https://mypod.store/admin. Error: The dataset admin doesn't exist
```

Looks like 3 of these accounts were never really created. B

**Comment by srosset81** [comment]
OK it seems all these accounts without datasets were created when the disk was full (around the 10th of October).

Normally they should have been deleted from the settings dataset. I [fixed](https://github.com/assemblee-virtuelle/semapps/commit/560506a7f28b2e4bbeccf921583452ee15b188dd) a bug in SemApps which prevented this.

I deleted the accounts manually.

For `effel` (an account created by Franciska @Laurin-W), I added back the missing containers, but there may be other problems.
